### PR TITLE
feat(core): add icons for expanding and collapsing Section Breaks

### PR DESCRIPTION
**Pull request by DeeMysterio**
expand-collapse-icons

### CODE REVIEW

File: frappe/public/js/frappe/form/toolbar.js [hunk 1]
```diff
@@ -183,18 +183,20 @@ frappe.ui.form.Toolbar = Class.extend({
 		}
 
 		// Expand all sections
-		let section_fields = me.frm.meta.fields.filter((field) => field.fieldtype == "Section Break" && field.collapsible == 1);
+		let section_fields = this.frm.meta.fields.filter((field) => field.fieldtype == "Section Break" && field.collapsible == 1);
 		if(section_fields.length != 0) {
 			this.page.add_menu_item(__("Expand Sections"), function () {
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js:188]: If we're doing icons now, do we still need the menu items?

File: frappe/public/js/frappe/form/toolbar.js [hunk 2]
```diff
@@ -183,18 +183,20 @@ frappe.ui.form.Toolbar = Class.extend({
...
-				for (let section of section_fields) {
-					me.frm.get_field(section.fieldname).collapse(false);
-				}
+				me.toggleSectionBreaks(section_fields, false);
 			}, true);
+			this.print_icon = this.page.add_action_icon("fa fa-angle-double-down fa-lg", function() {
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js]: You don't need to store it in a variable.

File: frappe/public/js/frappe/form/toolbar.js [hunk 3]
```diff
@@ -379,5 +381,14 @@ frappe.ui.form.Toolbar = Class.extend({
 		}
 
 		$(this.frm.wrapper).attr("data-state", this.frm.doc.__unsaved ? "dirty" : "clean");
+	},
+
+	toggleSectionBreaks: function(section_fields, collapse) {
+		let me = this;
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js]: This is not needed, since your scope is not changing in the loop. Simply use `this`.

**Review by Alchez — CHANGES_REQUESTED** [review]

File: frappe/public/js/frappe/form/toolbar.js (hunk 1 above)
Comment (DeeMysterio) [reply frappe/public/js/frappe/form/toolbar.js:188]: @neilLasrado says Yes. This is the same case as that of Print.

File: frappe/public/js/frappe/form/toolbar.js [hunk 4]
```diff
@@ -379,5 +381,13 @@ frappe.ui.form.Toolbar = Class.extend({
 		}
 
 		$(this.frm.wrapper).attr("data-state", this.frm.doc.__unsaved ? "dirty" : "clean");
+	},
+
+	toggleSectionBreaks: function(section_fields, collapse) {
+		for (let section of section_fields) {
+			this.frm.get_field(section.fieldname).collapse(collapse);
```
Comment (vjFaLk) [inline frappe/public/js/frappe/form/toolbar.js]: `collapse` is a bad name. I think it should be `collapse_state` or just `state`

**Review by vjFaLk — CHANGES_REQUESTED** [review]

**Review by vjFaLk — APPROVED** [review]

**Review by vjFaLk — APPROVED** [review]

**Comment by DeeMysterio** [comment]
Made this to [develop](https://github.com/frappe/frappe/pull/9025)

**Pull request merged**